www/docs/en/dev/config_ref/index.md

```diff
@@ -362,6 +365,8 @@ PreferredContentMode<br/>{% cdv_vartype string %} {% cdv_platform ios %} | *Defa
 InAppBrowserStatusBarStyle<br/>{% cdv_vartype string %} {% cdv_platform ios %} | *Default: default* <br/>  Set text color style of the StatusBar for InAppBrowser for iOS. Valid values are: `lightcontent` and `default`.
 SetFullscreen<br/>{% cdv_vartype boolean %} {% cdv_platform android %} | *Default: false* <br/>  Same as the Fullscreen parameter in the global configuration of this xml file. This Android-specific element is deprecated in favor of the global Fullscreen element, and will be removed in a future version.
 ShowTitle<br/>{% cdv_vartype boolean %} {% cdv_platform android %} | *Default: false* <br/>  Show the title at the top of the screen.
+SplashScreenBackgroundColor<br/>{% cdv_vartype string %} {% cdv_platform ios %} | Sets the app's splashscreen background color. Supports a four-byte hex value, with the first byte representing the alpha channel, and standard RGB values for the following three bytes.
```

Comment:
`SplashScreenBackgroundColor` should also be added to: https://github.com/apache/cordova-docs/blob/master/www/docs/en/dev/core/features/splashscreen/index.md